Sources/MintKit/Mint.swift

```diff
@@ -405,6 +405,43 @@ public class Mint {
         output("Installed \(packageCount) from \(mintFilePath.string)".green)
     }
 
+    public func dump(force: Bool) throws {
+
+        guard mintFilePath.exists == false || force else {
+            throw MintError.couldNotOverwriteMintfile(mintFilePath.string)
+        }
+
+        guard packagesPath.exists else {
+            output("No mint packages installed")
+            return
+        }
+
+        var packagesToExport: [String] = []
+
+        let packages: [Path] = try packagesPath.children().filter { $0.isDirectory }
+
+        try packages.forEach { (packagePath) in
+            let components = packagePath.lastComponent.split(separator: "_")
+            let repoOwner = String(components[components.count - 2])
+            let packageName = String(components.last!)
+            let versions = try (packagePath + "build")
+                .children()
+                .filter { !$0.lastComponent.hasPrefix(".") }
+                .sorted()
+                .map { $0.lastComponent }
+
+            for version in versions {
+                packagesToExport.append("\(repoOwner)/\(packageName)@\(version)")
```

Comment:
One issue here is that packages can be installed from all sorts of sources, like non github sites, or ssh only urls

Reply:
That's true, I will amend this PR in the following days to support also these cases. Thanks for the input!

Reply:
I changed the implementation to use the repo as saved in the metadata.json